src/components/organism/NowTodo.js

```diff
@@ -3,18 +3,26 @@ import { H1 } from "../atoms/H1";
 import { Span } from "../atoms/Span";
 import ToDoItem from "../molecules/ToDoItem";
 import { styled } from "styled-components";
+import { useRecoilValue, useRecoilState } from "recoil";
+import { nowTodoList, solvedTodoList } from "../../recoil/atom.ts";
 
-function NowTodo({ nowTodo, setSolvedTodo }) {
-  console.log(nowTodo);
+function NowTodo() {
+  const nowTodo = useRecoilValue(nowTodoList);
+  const [solvedToDo, setSolvedTodo] = useRecoilState(solvedTodoList);
   return (
     <StyledNowTodo>
       <H1>✔ 지금 당장 해결하세요.</H1>
       <Span color="var(--lightWhite)">
         총 {nowTodo.length} 개의 할 일이 있어요.
       </Span>
       <StyledList>
-        {nowTodo.map((todo) => (
-          <ToDoItem todo={todo} setSolvedTodo={setSolvedTodo} />
+        {nowTodo.map((todo, idx) => (
+          <ToDoItem
+            isSolved={false}
+            todo={todo}
+            setSolvedTodo={setSolvedTodo}
+            key={idx}
+          />
```

Comment:
map 함수 사용 시 key 를 index 로 설정하는 것은 권장하지 않는다고 합니다.
todo의 값으로 key를 설정하거나 todo의 내용이 중복될 수 있다면 todo+index 로 고유의 key값을 설정하는 것을 추천드립니다~!!
[key값으로 index를 단독으로 사용하면 안되는 이유](https://medium.com/sjk5766/react-%EB%B0%B0%EC%97%B4%EC%9D%98-index%EB%A5%BC-key%EB%A1%9C-%EC%93%B0%EB%A9%B4-%EC%95%88%EB%90%98%EB%8A%94-%EC%9D%B4%EC%9C%A0-3ce48b3a18fb)

```suggestion
{nowTodo.map((todo, idx) => (
          <ToDoItem
            isSolved={false}
            todo={todo}
            setSolvedTodo={setSolvedTodo}
            key={todo+idx}
          />